docs/UserGuide.md

```diff
@@ -273,7 +262,7 @@ Examples:
 
 ### Locating internships by: `find`
 
-Find internships whose data in the target category matches the given keyword/s.
+Looking for some internship applications? Find internships whose data in the target category matches the given keyword/s.
```

Comment:
some -> a specific

Reply:
was thinking of them but not just be one internship, not sure how to parse as well